static/src/js/actions/preferences.js

```diff
@@ -39,16 +41,20 @@ export const updatePreferences = data => (dispatch, getState) => {
 
       dispatch(updateAuthTokenFromHeaders(headers))
       dispatch(setPreferences(preferences))
-
       dispatch(setIsSubmitting(false))
+      addToast('Preferences saved!', {
```

Comment:
The test for `updatePreferences` should mock `addToast` and ensure that the correct parameters are passed for this update